core/http/src/tls/listener.rs

```diff
@@ -52,16 +60,15 @@ pub struct TlsStream {
     state: TlsState,
     certs: Certificates,
 }
-
```

Comment:
Why are these empty lines going away? We always want an empty line between unrelated items.